RubeesEat.IntegrationTests/OpenIDIntegrationTestBase.cs

```diff
@@ -0,0 +1,58 @@
+using Microsoft.AspNetCore.Hosting;
+using System.Collections.Immutable;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Hosting;
+using RubeesEat.IntegrationTests.WebTesting;
+using RubeesEat.Model;
+
+namespace RubeesEat.IntegrationTests;
+
+public class OpenIDIntegrationTestBase
```

Comment:
This type should be SeleniumIntegrationTestBase.
The SeleniumIntegrationTestBase should derive and replace / override relevant features and be named in a sensible way.

Reply:
Now i have the following inheritance structure:
SeleniumIntegrationTestBase > WithoutAuth > WithTestPerson > WithTestBills

I'm writing this here to make it easier to review since Github does not display the files in the order that makes sense for the inheritance.